src/visualizers/VisualizerHistogram.ts

```diff
@@ -0,0 +1,211 @@
+import {VisualizerExportModule} from '@/visualizers/VisualizerInterface'
+import {VisualizerDefault} from './VisualizerDefault'
+import {natlog} from '../shared/math'
+
+class HistogramVisualizer extends VisualizerDefault {
+    name = 'Histogram'
+
+    binSize = 1
+    terms = 10
+    firstIndex = 1
+    linear = false
+
+    params = {
+        binSize: {
+            value: this.binSize,
+            forceType: 'integer',
+            displayName: 'Bin Size',
+            required: true,
+        },
+
+        firstIndex: {
+            value: this.firstIndex,
+            forceType: 'integer',
+            displayName: 'First Index',
+            required: true,
+        },
+
+        terms: {
+            value: this.terms,
+            forceType: 'integer',
+            displayName: 'How many terms of the series',
+            required: true,
+        },
+    }
+
+    checkParameters() {
+        const status = super.checkParameters()
+
+        if (this.params.binSize.value < 1) {
+            status.isValid = false
+            status.errors.push('Bin Size can not be less than 1')
+        }
+
+        if (this.params.binSize.value < 1) {
+            status.isValid = false
+            status.errors.push('Bin Size can not be less than 1')
+        }
+
+        if (this.params.firstIndex.value < 1) {
+            status.isValid = false
+            status.errors.push('First Index can not be less than 1')
+        }
+
+        return status
+    }
+
+    //find the largest value of the sequence asked for
```

Comment:
Generally, we link to put spaces between the `//` and the comment.

Reply:
Addressed in https://github.com/numberscope/frontscope/pull/183/commits/e35d2e707ce523dcbb7e21a80fb63bd96b6671a0. I also made the comments have proper capitalization and punctuation since they were full sentences.